components/bl-double-calendar-component/component.json

```diff
@@ -0,0 +1,143 @@
+      "settingTooltip": "This handler allows you to specify a date selection range. This will work if you select a 'Specific date range' mode.",
+      "showInSettings": true,
+      "defaultValue": "7",
+      "hasLogicHandler": true,
+      "handlerId": "selectionRangeLogic",
+      "handlerLabel": "Selection range logic",
+      "handlerDescription": "This handler allows you to specify a date selection range. This will work if you select a 'Specific date range' mode.",
+      "dataBinding": true
+    },
+    {
+      "type": "text",
+      "name": "selectedDate",
+      "label": "Selected date",
+      "settingTooltip": "This handler allows you to specify the default selected date. Date format to be specified \"year/month/day\".",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "selectedDateLogic",
+      "handlerLabel": "Selected date logic",
+      "handlerDescription": "This handler allows you to specify the default selected date. Date format to be specified \"year/month/day\".",
+      "dataBinding": true
+    },
+    {
+      "type": "checkbox",
+      "name": "displayHeader",
+      "label": "Display header",
+      "settingTooltip": "This handler allows you to control the display of the header.",
+      "showInSettings": true,
+      "defaultValue": true,
+      "hasLogicHandler": true,
+      "handlerId": "displayHeaderLogic",
+      "handlerLabel": "Display header logic",
+      "handlerDescription": "This handler allows you to control the display of the header."
+    },
+    {
+      "type": "text",
+      "name": "dateFrom",
+      "label": "Date from",
+      "settingTooltip": "This handler allows you to specify a start date. Date format to be specified \"year/month/day\".",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "dateFromLogic",
+      "handlerLabel": "Date from logic",
+      "handlerDescription": "This handler allows you to specify a start date. Date format to be specified \"year/month/day\".",
+      "dataBinding": true
+    },
+    {
+      "type": "text",
+      "name": "dateTo",
+      "label": "Date to",
+      "settingTooltip": "This handler allows you to specify an end date. Date format to be specified \"year/month/day\".",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "dateToLogic",
+      "handlerLabel": "Date to logic",
+      "handlerDescription": "This handler allows you to specify an end date. Date format to be specified \"year/month/day\".",
+      "dataBinding": true
+    }
+  ],
+  "eventHandlers": [
+    {
+      "name": "onStartDateChange",
+      "label": "On start date change event",
+      "contextBlocks": [
+        {
+          "id": "startDate",
+          "label": "Start date"
+        },
+        {
+          "id": "daysAmount",
+          "label": "Days amount"
+        }
+      ]
+    },
+    {
+      "name": "onEndDateChange",
+      "label": "On end date change event",
```

Comment:
On End Date Change Even

Fix it everywhere. I don't know why, but all our handlers have written using a capital letters.